crates/proof_of_stake/src/lib.rs

```diff
@@ -1478,11 +1478,7 @@ where
 
     // Check that there are unbonded tokens available for withdrawal
     if unbond_handle.is_empty(storage)? {
-        return Err(WithdrawError::NoUnbondFound(BondId {
-            source: source.clone(),
-            validator: validator.clone(),
-        })
-        .into());
+        return Ok(token::Amount::zero());
```

Comment:
we can also remove the `WithdrawError` now

Reply:
done in 2858dbc7dd285b20d67c4c4d10c99ff2ec87305f